presqt/api_v1/helpers/validation.py

```diff
@@ -57,4 +57,4 @@ def token_validation(request):
     except KeyError:
         raise PresQTAuthorizationError(
             "'presqt-source-token' missing in the request headers.",
-            status.HTTP_400_BAD_REQUEST)
\ No newline at end of file
+            status.HTTP_401_UNAUTHORIZED)
```

Comment:
I think 400 is actually correct based on what you are doing here.
You are just seeing if it was provided in the request, not whether it is valid.

Right?